src/actions/roles.js

```diff
@@ -14,9 +14,6 @@ import type {Role} from 'types/roles';
 export function getRolesByNames(rolesNames: Array<string>) {
     return bindClientFunc({
         clientFunc: Client4.getRolesByNames,
-        onRequest: RoleTypes.ROLES_BY_NAMES_REQUEST,
```

Comment:
i checked across webapp and mobile and though we have references in  https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/permission_schemes_settings/permission_system_scheme_settings/index.jsx#L21 and https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/index.jsx#L30. We are not using them so we can remove them from webapp. Can you make a PR to remove them?

Reply:
@sudheerDev
I picked up https://github.com/mattermost/mattermost-server/issues/12520 and will be building off of this PR.
I opened up https://github.com/mattermost/mattermost-webapp/pull/7562 that removes `getRolesByNames` from the files you listed.